nodejs/v3/remix/app/processmon.toml

```
@@ -0,0 +1,15 @@
+[[paths_to_watch]]
+path = "/app"
```

```suggestion
path = "/app"
ignore = ["build", "public/build"]
```
If processmon watches those paths it immediately restarts the app after start (and modification of those directories), then I get the error:

```
remix-app-1  | node: Error: listen EADDRINUSE: address already in use :::4001
remix-app-1  | node:     at Server.setupListenHandle [as _listen2] (node:net:1751:16)
remix-app-1  | node:     at listenInCluster (node:net:1799:12)
remix-app-1  | node:     at Server.listen (node:net:1887:7)
remix-app-1  | node:     at Function.listen (/app/node_modules/express/lib/application.js:635:24)
remix-app-1  | node:     at Object.<anonymous> (/app/node_modules/@remix-run/serve/dist/cli.js:48:84)
remix-app-1  | node:     at Module._compile (node:internal/modules/cjs/loader:1256:14)
remix-app-1  | node:     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
remix-app-1  | node:     at Module.load (node:internal/modules/cjs/loader:1119:32)
remix-app-1  | node:     at Function.Module._load (node:internal/modules/cjs/loader:960:12)
remix-app-1  | node:     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
```